clarity/src/vm/functions/database.rs

```diff
@@ -1069,7 +1069,7 @@ pub fn special_get_tenure_info(
     };
 
     let current_height = env.global_context.database.get_current_block_height();
-    if height_value >= current_height {
+    if height_value > current_height {
```

Comment:
This is a consensus-breaking change.  cc @kantai @obycode @saralab

Reply:
yes, it is, but I think it should make it in, otherwise it is surprising behavior. I'm going to finish the tests for this PR tonight.

Reply:
Yes agreed.  Just wanted to make sure that this was flagged for all of the testnets -- we can't just upgrade the nodes; we'll need to wipe the chainstate as well.